core/arch/arm/kernel/secure_partition.c

```diff
@@ -603,6 +603,71 @@ static TEE_Result handle_fdt_dev_regions(struct sp_ctx *ctx, void *fdt)
 	return TEE_SUCCESS;
 }
 
+static TEE_Result swap_sp_endpoints(uint32_t endpoint_id,
+				    uint32_t new_endpoint_id)
+{
+	struct sp_session *session = sp_get_session(endpoint_id);
+	TEE_Result res = TEE_SUCCESS;
+	uint32_t manifest_endpoint_id = 0;
+
+	/*
+	 * We don't know in which order the SPs are loaded. It can
```

Comment:
You can drop "It can be that" from the second sentence.

Reply:
Ack